aviary/variable_info/variable_meta_data.py

```diff
@@ -972,6 +1020,18 @@
     desc='Drag polar computed during Aviary pre-mission.',
 )
 
+add_meta_data(
+    Aircraft.Design.EMERGENCY_MASS,
```

Comment:
@crecine does GASP have any additional information on what's included in emergency mass? Design is an ok place to put it if it doesn't neatly fit into any of our other categories

Reply:
I'll see if I can find any more detailed information, but it's the cabin emergency equipment (fire extinguishers, oxygen tanks, smoke hoods, etc)